### PR TITLE
Add delayBeforeCheck parameter.

### PR DESCRIPTION
**Pull request by kunickiaj**
**What this PR does / why we need it**:
Add delayBeforeCheck parameter.

This is helpful in the scenario where you want to use dns-01
challenges in a public route53 zone, but your cluster's
resolver is using a private route53 zone and the initial TXT
check fails. This allows you to skip the check and instead
use a fixed delay before the ACME verification happens.

### CODE REVIEW

**Comment by dtomcej** [comment]
Can you put that in an `if` block, so that it is not specified if it is not included in the `values.yaml`?

**Comment by davidkarlsen** [comment]
@kunickiaj ping?

**Comment by kunickiaj** [comment]
Just posted an update. Thanks for the reminder!

**Comment by kunickiaj** [comment]
I think the edited approval tag did not trigger tide.

File: stable/traefik/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.39.1
+version: 1.40.0
```
Comment (davidkarlsen) [inline stable/traefik/Chart.yaml]: can you bump the version again - something was merged before this

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/traefik/Chart.yaml (hunk 1 above)
Comment (davidkarlsen) [reply stable/traefik/Chart.yaml]: @kunickiaj

**Pull request merged**